lib/services/rocket.py

```
@@ -219,67 +211,19 @@ def get_rocketpy_parachute(parachute: Parachute) -> RocketPyParachute:
-        # Restricting access to functions or attributes
-        for node in ast.walk(lambda_node):
-            if isinstance(node, ast.Call):
-                raise InvalidParachuteTrigger(
-                    "Calling functions is not allowed in the expression."
-                ) from None
-            if isinstance(node, ast.Attribute):
-                raise InvalidParachuteTrigger(
-                    "Accessing attributes is not allowed in the expression."
-                ) from None
-        return True
+        return False
```

Ruff

Return the condition `bool(isinstance(trigger, (int, float)))` directly

Replace with `return bool(isinstance(trigger, (int, float)))`

(SIM103)